libraries/default.rb

```diff
@@ -736,14 +736,16 @@ def install_opencast_log_configuration(current_deploy_root)
       oc_log_dir = node.fetch(:opencast_log_directory, '/var/log/opencast')
       oc_log_level = node.fetch(:opencast_log_level, 'INFO')
       dce_log_level = node.fetch(:dce_log_level, 'DEBUG')
+      hostname = node[:opsworks][:instance][:hostname]
       template %Q|#{current_deploy_root}/etc/org.ops4j.pax.logging.cfg| do
```

Comment:
Where does the "node[:opsworks][:instance][:hostname]" value come from? Is that something different than the cluster config?

Reply:
`node` is an internal chef data structure. it's available to all recipes.